conf/appsettings.json.template

```diff
@@ -129,5 +129,6 @@
     "DuplicationDetectionEnabled": "true" // if enabled, the same bundle (md5 hash check) does not get analyzed twice
     "DownloadServiceRetry": "5", // Number of retries when trying to download a dump file from an URL
     "DownloadServiceRetryTimeout": "500" //Time between download retries in milliseconds
+    "DownloadServiceHttpClientTimeout": 4 //HttpClient timeout in minutes
```

Comment:
We should be clearer about units in the names of settings for timespans.

 - "JiraBundleSearchDelay": "0.5", //Time to wait (in minutes) ...
 - "JiraBundleSearchTimeSpan": "30.00:00:00", //Maximum age of bundle for issue search
 - "DownloadServiceRetryTimeout": "500" //Time between download retries in milliseconds
 - "DownloadServiceHttpClientTimeout": 4 //HttpClient timeout in minutes

pretty bad mixture.

I'd propose:
 - append unit to setting name
 - ideally, just use TimeSpan datatype everywhere

Not sure if we should do this for all settings right now (maybe in a separate issue. it will have upgrade-issues). But at least, use TimeSpan for the new setting here.

Reply:
Thanks for the feedback!
I changed the type for the new setting.